tests/aws/conftest.py

```diff
@@ -36,7 +37,12 @@
 
 
 @pytest.hookimpl()
-def pytest_configure(config):
+def pytest_configure(config_):
```

Comment:
Seems like pytest does not like this renaming. We might want to rename the localstack config in the import instead.